CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_message_files(
    MasterboardDataMsg.msg
    RobotModeDataMsg.msg
    ToolDataMsg.msg
+   VersionInformation.msg
```

Comment:
could you motivate why you create a separate message and return that in the service result instead of making the service result what is now the body of `VersionInformation.msg`?

Reply:
This is the structure used in the dashboard messages within the Universal Robots ROS driver. I can remove the message again and move it in to the service instead.